lib/uuidable/active_record.rb

```diff
@@ -24,6 +24,8 @@ module ClassMethods
       def uuidable(as_param: true) # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
         # Configure all uuid columns for MySQL. Database may not be connected (i.e. on assets precompile), so we must supress errors.
         conn_config = respond_to?(:connection_db_config) ? connection_db_config.configuration_hash : connection_config
+        return unless schema_loaded?
```

Comment:
If I understand correctly, the main problem is in `columns` call? Let's add this check to the `if` below and do not return completely from this method when schema is not loaded.

Reply:
fixed